src/frontends/paddle/src/op/interp.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include "default_opset.hpp"
 #include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/opsets/opset4.hpp"
```

Comment:
How about upgrade to latest opset too?

Reply:
> How about upgrade to latest opset too?

`Interpolate` of OpenVINO opset4 can match PDPD's `Interpolate` directly.
But `Interpolate` of the latest opset needs more extra ops to map such op.